Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -70,7 +75,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $em = $doctrine->getManager($input->getOption('em'));
 
         if (!$input->getOption('append')) {
-            $ui->ask('Careful, database will be purged. Do you want to continue y/N ?', false);
+            if (!$ui->confirm('Careful, database will be purged. Do you want to continue?', false)) {
```

Comment:
Why did you change the previous options? yes/non instead of y/N?

Reply:
It was broken before :) - if you said “n” it continued anyways. I fixed it (and also changed to the standard confirm)

Reply:
Correct, this was broken in #220 which hasn't been released yet. I've extracted the fix to #263 to have it fixed in 3.0 as well since this PR will target 3.1.